src/scenes/Reports/ReportsList.tsx

```diff
@@ -201,7 +201,7 @@ export default function ReportsList({ fromConsole }: ReportsListsProps): JSX.Ele
       fuzzySearchColumns={fuzzySearchColumns}
       id='accelerator-reports-table'
       isClickable={() => false}
-      Renderer={ReportCellRenderer}
+      Renderer={ReportCellRenderer({ projectId: urlProjectId })}
```

Comment:
This will need to be updated to get the project id from participant project data on the participant side, but if you’d rather not deal with it, I can handle it when I add the participant-side report view.

Reply:
Yes, we can add it when working with participant side